OpenRPA.NativeMessagingHost/myAddon2/content.js

```diff
@@ -709,7 +709,12 @@ if (true == false) {
                         // https://stackoverflow.com/questions/3437786/get-the-size-of-the-screen-current-web-page-and-browser-window
                         let message = { functionName: action, frame: frame, parents: 0, xpaths: [] };
                         let targetElement = null;
-                        targetElement = event.target || event.srcElement;
+                        try {
+                            targetElement = event.composedPath()[0];
+                        } catch (e) {
+                            console.error('event composed path');
+                        }
+                        targetElement = targetElement || event.target || event.srcElement;
```

Comment:
Is composedPath supported both in Chrome and Edge? If it's supported is always an array with one element? I'm asking this because I mostly prefer a more "safe" approach, avoiding a try/catch (that should be there, just in **exceptional** case)

Reply:
I think so, our concern is the events, we have different kinds of events, not sure all of them can be used like this, also the user can create their on customEvents, if we can ensure it's will work for all I think will be safe to remove the try-catch

Reply:
We can also use like: 
```
if(event.isComposed)
    targetElement = event.composedPath()[0];
```
Since it can be different just if the event is of the type composed

Reply:
Yes, it's supported for both Chrome and Edge - https://developer.mozilla.org/en-US/docs/Web/API/Event/composedPath

The array's length varies depending on the event's propagation. It returns all elements that the event propagated to, and the first element is the one that originates the event.

The try/catch is because there are some cases where the current event is not composed, which means the composedPath() function won't work

Reply:
Can you apply this change instead of using the try-catch
```
if(event.isComposed)
    targetElement = event.composedPath()[0];
```